components/filter-field/src/filter-field-tag/filter-field-tag.spec.ts

```diff
@@ -38,11 +38,10 @@ import { Platform } from '@angular/cdk/platform';
 import {
   createComponent,
   MockNgZone,
-} from '@dynatrace/barista-components/testing';
-import {
-  mockObjectProperty,
   mockGetComputedStyle,
-} from '@dynatrace/barista-components/testing/mock';
+} from '@dynatrace/barista-components/testing/browser';
+
+import { mockObjectProperty } from '@dynatrace/barista-components/testing/node';
```

Comment:
this is not ideal to import inside a component from testing/node...

Reply:
Its the spec file ;)